glean-core/src/ac_migration/mod.rs

```diff
@@ -0,0 +1,108 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+//! A module containing glean-core code for supporting data migration
+//! from glean-ac.
```

Comment:
IMO we should document what it migrates and also call out that this is definitely temporary while we transition Glean AC and planned to be removed in 2020